examples/postgresql-multiple-datasources/helm.yaml

```diff
@@ -0,0 +1,53 @@
+build:
+  uri: https://github.com/wildfly/wildfly-s2i
+  mode: s2i
```

Comment:
Our existing examples need some cleanup. We could cleanup this new one. you can remove this field, that is the default value.

Reply:
Done, removed these items as they are the default values.